src/containers/cellPairTraversals/SlicedTraversal.h

```diff
@@ -95,37 +97,14 @@ inline void SlicedTraversal<ParticleCell, CellFunctor>::rebuild(std::vector<Part
   locks.resize(numSlices);
 }
 
-// FIXME: Remove this as soon as other traversals are available
-template <class ParticleCell, class CellFunctor>
-inline void SlicedTraversal<ParticleCell, CellFunctor>::traverseCellPairsFallback() {
-  std::array<unsigned long, 3> endid;
-  for (int d = 0; d < 3; ++d) {
-    endid[d] = this->_cellsPerDimension[d] - 1;
-  }
-  for (unsigned long z = 0; z < endid[2]; ++z) {
-    for (unsigned long y = 0; y < endid[1]; ++y) {
-      for (unsigned long x = 0; x < endid[0]; ++x) {
-        unsigned long ind = ThreeDimensionalMapping::threeToOneD(x, y, z, this->_cellsPerDimension);
-        this->processBaseCell(ind);
-      }
-    }
-  }
-}
-
-template <class ParticleCell, class CellFunctor>
-inline void SlicedTraversal<ParticleCell, CellFunctor>::traverseCellPairs() {
+template <class ParticleCell, class PairwiseFunctor, bool useSoA, bool useNewton3>
+inline void SlicedTraversal<ParticleCell, PairwiseFunctor, useSoA, useNewton3>::traverseCellPairs(
+    std::vector<ParticleCell> &cells) {
   using std::array;
 
   auto numSlices = _sliceThickness.size();
   // 0) check if applicable
 
-  // use fallback version if not applicable
-  // FIXME: Remove this as soon as other traversals are available
-  if (not isApplicable()) {
```

Comment:
maybe throw an exception if it isn't applicable. just to prevent it being called when it isn't applicable

Reply:
Applicability check is now here:
https://github.com/AutoPas/AutoPas/blob/75d11a55f2b192119544169fee097a1be4d0a511/src/selectors/TraversalSelector.h#L108

So it cannot be called when not applicable.